robot-server/robot_server/service/session/session_types/check_session.py

```diff
@@ -75,7 +72,7 @@ async def create(
             calibration_check = CheckCalibrationUserFlow(
                 configuration.hardware,
                 has_calibration_block=has_calibration_block,
-                tip_rack_defs=[cast("LabwareDefinition", rack) for rack in tip_racks],
+                tip_rack_defs=[rack for rack in tip_racks],
```

Comment:
I know this becomes pretty redundant without the `cast`, but I felt it was ever so slightly safer to leave this alone just in case the clone-via-list-comprehension was important

Reply:
Agreed. In the spirit of [this Twitter thread that I like a lot](https://twitter.com/mcclure111/status/1274422825831596039), can we do this?

```suggestion
                # Clone-via-list-comprehension inherited from prior code; not sure if it matters.
                tip_rack_defs=[rack for rack in tip_racks],
```

Reply:
Added a TODO with a call-to-action in case someone with more historical knowledge than me is touching this file in the future